.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
Do we need this `_` for gitignore?

Reply:
This was autogenerated by husky when I installed using `yarn`. This folder contains the husky script that's installed based on your OS. `.sh` for MacOS / `.bat` for Windows.

It looks like it's intended 👍🏼